src/sql/workbench/browser/modelComponents/splitviewContainer.component.ts

```diff
@@ -92,6 +99,25 @@ export default class SplitViewContainerImpl extends ContainerBase<FlexItemLayout
 		return basicView;
 	}
 
+	@debounce(300)
+	private resizeSplitview() {
+		if (this._resizeable) {
+			this._splitViewSize = this.calculateSplitViewSize(this.orientation);
+			this._splitView.layout(this._splitViewSize);
+		}
+	}
+
+	/**
+	 * Calculates the size of the split view based on the dimensions of the model view container and orientation of the splitview
+	 * @param orientation
+	 * @returns
+	 */
+	private calculateSplitViewSize(orientation: string): number {
+		const modelViewContainer = document.getElementsByClassName('model-view-container')[0] as HTMLDivElement;
```

Comment:
Doing a global search for all model-view-containers seems risky - we could easily have multiple model view containers (for example if the user has split editor panes).

Probably better to use dom.findParentWithClass to find the parent container of this specific splitview.